packages/manager/apps/ips/src/pages/onboarding/index.tsx

```diff
@@ -38,25 +49,41 @@ export default function Onboarding() {
     },
   ];
 
-  const title: string = t('title');
-  const description: string = t('description');
   const imgSrc = {
     src: onboardingImgSrc,
   };
 
   return (
-    <OnboardingLayout
-      title={title}
-      img={imgSrc}
-      description={description}
-      orderButtonLabel={t('orderButtonLabel')}
-      orderHref={t('orderButtonLink')}
-      moreInfoButtonLabel={t('moreInfoButtonLabel')}
-      moreInfoHref={t('moreInfoButtonLink')}
-    >
-      {tileList.map((tile) => (
-        <Card key={tile.id} href={tile.href} texts={tile.texts} />
-      ))}
-    </OnboardingLayout>
+    <>
+      <Breadcrumb />
```

Comment:
There are no Breadcrumb on Onboarding page
Usually we are redirected to this page when there is no service yet so there is no other page to view with the breadcrumb